### DISCUSSION TITLE
Installing on linux without docker

### DISCUSSION THREAD

**Discussion by ValentinDoche** [cpvalente/ontime#884] [Q&A]
Hello, 

I'm looking to install Ontime without docker. I plan to run it on a proxmox in an LXC container but I don't want to have a Docker in my container. 

I've tried to install it manually, but I'm not good enough to understand why I get errors when I build the client and the server. 

I think it would be a plus to have a manual installation part in the wiki.

Thanks to all and have a nice day

**Comment by cpvalente** [comment]
Hi @ValentinDoche, thank you for reaching out,

If I understand correctly, you are trying to build the app yourself to consume in a LXC container?
Nice

We dont have instruction specifically for building this but you can, for the most part, take it from how the docker file is done. See the relevant documentation 
https://github.com/cpvalente/ontime/blob/master/DEVELOPMENT.md
https://github.com/cpvalente/ontime/blob/master/Dockerfile

Although the building of the  app itself shouldnt be particularly cumbersome. You would need to make your setup mimick a docker install. This would mean replicating folder structure and environment variables
https://github.com/cpvalente/ontime/blob/master/Dockerfile#L5

This is again, not particularly difficult, but I understand that it can be tricky to get right

Easiest way forwards for now is if you could share with us the issues you are finding and we will eventually figure it out together

Hope this helps
